### DISCUSSION TITLE
Firedrake not Working After Installation

### DISCUSSION THREAD

**Discussion by afeisede** [firedrakeproject/firedrake#3481] [Firedrake support]
Hi everyone!
I just installed firedrake through Homebrew. I was able to activate the venv but I cannot import Firedrake in Python neither was I able to run the Test Suite using (pytest tests/regression/ -k "poisson_strong or stokes_mini or dg_advection").
I have attached the firedrake-install.log, configure.log, and make.log files.
Please I need help! I use a mac os.
[firedrake-install.log](https://github.com/firedrakeproject/firedrake/files/14793923/firedrake-install.log)
[configure.log](https://github.com/firedrakeproject/firedrake/files/14793937/configure.log)

**Comment by connorjward** [comment]
There are two things to flag:
1. You are using the wrong Python to try and install Firedrake. You should use a Python installed with homebrew (`$(brew --prefix)/bin/python3`).
2. We are currently experiencing build issues with PETSc with the latest version of Xcode ([discussion](https://github.com/firedrakeproject/firedrake/discussions/3465#discussioncomment-8965654)). This doesn't seem to be the issue that you're hitting but I can't be totally sure.

I think that there are a couple of things to try:
1. Make sure that homebrew is updated and try running the install using the correct Python.
2. If this doesn't work then try [updating your Xcode](https://xcodereleases.com/) (ideally to 15.2 instead of 15.3 as we are having issues with 15.3). Run the install again.
3. If that still doesn't work, and you are using 15.3, then please see the workaround [here](https://github.com/firedrakeproject/firedrake/discussions/3465#discussioncomment-8965654).
